spring-integration-sftp/src/test/java/org/springframework/integration/sftp/dsl/SftpTests.java

```diff
@@ -117,12 +119,14 @@ public void testSftpInboundStreamFlow() throws Exception {
 		assertNotNull(message);
 		assertThat(message.getPayload(), instanceOf(InputStream.class));
 		assertThat(message.getHeaders().get(FileHeaders.REMOTE_FILE), isOneOf(" sftpSource1.txt", "sftpSource2.txt"));
+		StreamUtils.copy((InputStream) message.getPayload(), new ByteArrayOutputStream());
```

Comment:
Right that's what I am saying; on my Windows VM, the file.delete() was successful (I didn't get the "couldn't delete" message - that's why I added the additional "Contents" log when the directory delete fails.